src/format/opamPath.ml

```diff
@@ -117,6 +117,10 @@ module Switch = struct
 
   let sources_dir t a = meta t a / "sources"
 
+  let extra_files_dir t a = sources_dir t a / "extra-files"
```

Comment:
We should choose another directory: this might conflict with a pinned package called "extra-files".